### DISCUSSION TITLE
A Missing Module

### DISCUSSION THREAD

**Discussion by jacktshishonge** [microsoft/pylance-release#3418] [Q&A]
Hey guys I have been using Visual Studio Code  for a while now and I can see that there are certain modules missing when i try 
to import them using python. Like
beautifulsoup4, requests, and lxml 
if anyone has any knowledge on this please help as soon as possible

**Comment by LIPDProductionsInc** [comment]
To tack onto this, is there a way we can suggest modules to be added as a supported one in Pylance?

**Comment by uOWISIMOn** [comment]
**_numpy_** also can't be imported

**Reply by EnzoMeirino** [reply]
im with the same problem

**Comment by r3m0t** [comment]
Visual Studio Code doesn't come with Python included. You need to install it separately through the Microsoft Store, a distribution like Anaconda, or from the Python.org official website.

Then install any missing packages with pip or if you're using Conda, conda.

Here's a guide I found: https://projects.raspberrypi.org/en/projects/using-pip-on-windows/2

**Reply by LIPDProductionsInc** [reply]
That's not the issue. I have modules like `psutil` and `dotenv` installed and Pylance doesn't resolve them

**Reply by r3m0t** [reply]
That's a different issue then. Have a look here - https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md

**Reply by debonte** [reply]
@r3m0t, please file an issue on this.
